osu.Game/Screens/Play/DimmableStoryboard.cs

```diff
@@ -46,9 +51,15 @@ private void initializeStoryboard(bool async)
             drawableStoryboard = storyboard.CreateDrawable();
 
             if (async)
-                LoadComponentAsync(drawableStoryboard, Add);
+                LoadComponentAsync(drawableStoryboard, onStoryboardCreated);
             else
-                Add(drawableStoryboard);
+                onStoryboardCreated(drawableStoryboard);
+        }
+
+        private void onStoryboardCreated(DrawableStoryboard storyboard)
```

Comment:
small nit:
```csharp
        private void addStoryboard(DrawableStoryboard storyboard)
```

Reply:
This method doesn't only add the storyboard as a child, so I'd say that name would be misleading.

Reply:
I mean, it adds the storyboard, and also adds the overlay layer loaded by the storyboard to the needed container, doesn't sound misleading to me?

Reply:
To me `onStoryboardCreated` sounds like a callback (which it is), whereas `addStoryboard` sounds like something that should be called by the user.

I think it makes sense as is. In the scope of this file it doesn't make much of a difference.